app/models/database.py

```diff
@@ -73,11 +73,39 @@ def check_if_key_exists(self, key_value, is_user=False) -> bool:
     # READ
 
     def get_link(self, short_url: str) -> Link:
-        pass
         """
-        this function will take a shortlink as a key, and return the link object as the value
+        this function will take a shortlink as a key, and return the link
         """
+        try:
+            #look for the key
+            res = self.table.get_item(Key={'short_url': short_url})
+            if "Item" in res:
+                #Since the url exists, and this is a redirect request, increment the counter
+                self.increment_click(short_url)
+
+                #return the original url for forwarding
+                return res['Item']["original_url"]
```

Comment:
The return value above says "Link" but here you return the original URL. I think you'll want to return the Link here and then use that Link by whichever function calls this.